frontend/src/features/properties/map/MapView.scss

```diff
@@ -81,3 +81,8 @@ $narrowSidebarSize: 700px;
 .building-cursor .map .leaflet-container.leaflet-grab .leaflet-interactive {
   cursor: url('../../../assets/images/marker-icon-1x-blue.png') 12.5 41, default;
 }
+
+.map .leaflet-marker-icon.leaflet-interactive div,
+.map .leaflet-marker-icon.leaflet-interactive span {
+  pointer-events: none;
```

Comment:
prevent pointer events on the nested inner div/span in the cluster components. this ensures that the parent component (the component that is properly wired up in leaflet) ends up handling the click.